java-backend/src/test/java/org/kframework/backend/java/kil/BuiltinListTest.java

```diff
@@ -27,10 +27,6 @@
 
     @Before
     public void setUp() {
-        when(globalContext.getDefinition()).thenReturn(definition);
-        when(definition.subsorts()).thenReturn(subsorts);
-        when(subsorts.isSubsortedEq(Sort.LIST, Sort.LIST)).thenReturn(true);
-        when(subsorts.isSubsortedEq(Sort.LIST, Sort.INT)).thenReturn(true);
```

Comment:
Why is this not needed anymore?

Reply:
It was already not needed, but the update to the mockito version caused it to raise an error about unused stubs.